server/db/models/product.js

```diff
@@ -0,0 +1,29 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Product = db.define('product', {
+  name: {
+    type: Sequelize.STRING,
+    unique: true,
```

Comment:
I think unique should be false because we could have many torches called 'basic torch'